pkg/plugin/plugin.go

```diff
@@ -84,10 +85,24 @@ func (j *JiraPlugin) Validate(ctx context.Context, req *jvspb.ValidateJustificat
 		}, fmt.Errorf("failed to validate justification: %w", err)
 	}
 
+	// There is only one JQL and one issueKey, so the first match result is
+	// checked directly.
+	if len(result.Matches[0].MatchedIssues) == 1 {
+		issueID := strconv.Itoa(result.Matches[0].MatchedIssues[0])
+		baseURL := "https://verily-okta-sandbox.atlassian.net/browse/"
+		issueURL := baseURL + req.Justification.Value
+		return &jvspb.ValidateJustificationResponse{
+			Valid:   true,
+			Warning: result.Matches[0].Errors,
+			Annotation: map[string]string{
+				"jira_issue_id":  issueID,
+				"jira_issue_url": issueURL,
```

Comment:
Make the keys consts. Also I wonder if we can save "jira_" since it's already assumed in the context.

Reply:
I made them consts. As I observed that all the items in the config start with "jira_", I have retained the "jira_" here to maintain consistency.